deploy/examples/v2/process_migration.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: app.kiegroup.org/v2
+kind: KieApp
+metadata:
+  name: process-migration
+spec:
+  environment: rhpam-trial
+  objects:
+    processMigration:
+      image: "rhpam-process-migration-rhel8"
```

Comment:
why is this here? isn't this the default image?

Reply:
Removed